### PR TITLE
Add a Kotlin section and libraries with significant Kotlin support to it.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -629,7 +642,7 @@ <h2 class="module-name">Tweets</h2>
               <li class="profile-module">
                 <h2 class="module-name">Join us</h2>
                 <ul class="module-list">
-                  <li><a href="https://developer.squareup.com/blog/">Engineering Blog</a> &ndash; developer.squareup.com/blog</li>
+                  <li><a href="http://corner.squareup.com">Engineering Blog</a> &ndash; corner.squareup.com</li>
```
Comment (zach-klippenstein) [inline index.html]: This shouldn't be here.

File: repos.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
     "toggle": null,
     "Valet": ["iOS"],
     "wire": ["Android", "Java"],
-    "workflow": ["Android", "iOS"]
+    "workflow": ["Android", "iOS", "Kotlin"]
```
Comment (rjrjr) [inline repos.json]: Should we add this tag to leakcanary and sqldelight at the same time?

Comment (rjrjr) [reply repos.json]: cc @AlecStrong @pyricau @swankjesse

File: repos.json [hunk 3]
```diff
@@ -25,11 +25,12 @@
     "jirafy": ["Other"],
     "js-jose": ["JavaScript"],
     "Keywhiz": ["Java"],
-    "leakcanary": ["Android"],
+    "KotlinPoet": ["Kotlin"],
+    "leakcanary": ["Android", "Kotlin"],
```
Comment (zach-klippenstein) [inline repos.json]: @pyricau Added LeakCanary because of the heap parser.

File: repos.json [hunk 4]
```diff
@@ -38,21 +39,21 @@
...
     "rails-auth": ["Ruby"],
-    "retrofit": ["Android", "Java"],
+    "retrofit": ["Android", "Java", "Kotlin"],
     "seismic": ["Android"],
     "shift": ["Other"],
     "shuttle": ["Other"],
     "spacecommander": ["iOS"],
     "spoon": ["Android"],
-    "sqldelight": ["Android"],
+    "sqldelight": ["Android", "Kotlin"],
```
Comment (zach-klippenstein) [inline repos.json]: @alecholmes Should we add this to iOS too, since it's MPP?

Comment (AlecStrong) [reply repos.json]: yeeeeee

Comment (zach-klippenstein) [reply repos.json]: Done.

**Pull request merged**